__tests__/app.test.js

```diff
@@ -241,6 +241,111 @@ describe('Articles', () => {
 
     }); // Describe: GET /api/article/:article_id
 
+    describe('GET /api/articles/:article_id/comments', () => {
+        
+        test('should respond with an array of comments', () => {
+            // arrange
+            const articleId = 1;
+            // act
+            return request(app)
+            .get(`/api/articles/${articleId}/comments`)
+            .expect(200)
+            .then((response) => {
+                // assert
+                expect(response.body).toHaveProperty('comments');
+                expect(response.body.comments).toBeInstanceOf(Array);
+            });
+        });
```

Comment:
This test isn't really needed as your below test demonstrates that there s a comments property whose value is an array